### DISCUSSION TITLE
flag --endpoint-url

### DISCUSSION THREAD

**Discussion by 1g0rkR** [cargo-lambda/cargo-lambda#597] [Q&A] [answered]
Can somebody please specify which custom endpoint URL to target I can mention there in details? Thanks.

**Comment by calavera** [comment, marked as answer]
That flag is used if you want to send requests to a specific AWS region, or local testing tools, like LocalStack. See this commit for more details: https://github.com/cargo-lambda/cargo-lambda/commit/77bc628e47257a200a907c2c0c01ba2dbf28d06e
